src/components/GeoMap.js

```diff
@@ -473,7 +472,7 @@ export default class GeoMap {
         } else {
           const originalValueRaw = d.rollupValue[attrKey];
           const originalValue = (attrValue.transform) ?
-              attrValue.transform(originalValueRaw) : originalValueRaw;
+              attrValue.transform(originalValueRaw, this.maptable.rawData) : originalValueRaw;
```

Comment:
Oh spoke to quickly, that's useful indeed to calculate the percentile. Yep Maybe you should add an example to understand why we're passing the raw dataset :)